src/languageProvider/locales/ko-KR.json

```diff
@@ -212,7 +210,6 @@
   "str.to": "수신",
   "str.token": "토큰",
   "str.transaction": "트랜젝션",
-  "str.transactionId": "트랜젝션 ID",
```

Comment:
same here.

Reply:
I replaced it with `"str.transactionIdX": "Transaction ID: {txid}",`